styleguide/components/breadcrumbs/breadcrumbs.stories.mdx

```diff
@@ -11,11 +11,21 @@ import templateNoCollapse from './_no-collapse.html.haml';
 
 # Breadcrumbs
 
-Breadcrumbs are used at the top of pages to provide navigation and location context. There are two variants.
+Breadcrumbs show users the current location on the website and enable them to quickly move up to a parent level.
 
-## Default (collapse)
+## When to use
 
-By default they will collapse to the parent page on mobile. You can explicity set this behaviour by passing in
+Use breadcrumbs when a page is located in a hierarchy of at least two levels. 
+
+## When not to use
+
+Do not use breadcrumbs on a website with a flat hierarchy or to show the journey through a transaction. If a page is taking users through a multistep process, use the "Step-by-step sidebar navigation" (or consider if a progress indicator is needed) instead.
+
+## How it works
+
+Breadcrumbs should be placed on the top left of each page and below "Global navigation". They should start with a link to the home page and end with the current page title. Page titles of parent pages should be linked; the current should not have a link.
```

Comment:
If you are using the provided template in Rails "the current should not have a link" this will happen for you.